### PR TITLE
Discard filters test

### PR DESCRIPTION
**Pull request by anastasia8755**

### CODE REVIEW

File: src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductTypePage.java [hunk 1]
```diff
@@ -132,4 +133,15 @@ public int getProductsQuantity() {
 
         return Integer.parseInt(productsQuantity[1]);
     }
+
+    @Step("ProductTypePage: discard all selected filters")
+    public ProductTypePage discardAllFilters() {
+        $x("//button[@class='catalog-selection__link catalog-selection__link_type_reset']").shouldBe(Condition.enabled).click();
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductTypePage.java:139]: ??

File: src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductTypePage.java [hunk 2]
```diff
@@ -132,4 +133,15 @@ public int getProductsQuantity() {
...
+        return new ProductTypePage();
+    }
+
+    @Step("ProductTypePage: get discarding button display status")
+    public boolean isDiscardingButtonVisible() {
+        return isElementInvisible("//button[@class='catalog-selection__link catalog-selection__link_type_reset']");
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductTypePage.java:145]: is it visible or not?

File: src/main/java/com/ss/ita/rozetka/pageobject/utils/PageUtil.java [hunk 3]
```diff
@@ -19,4 +19,14 @@ public static boolean isElementVisible(String elementXPath) {
             return false;
         }
     }
+
+    public static boolean isElementInvisible(String elementXPath) {
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/utils/PageUtil.java:23]: don't we have ElementUtil or some thing for things like that?

File: src/test/java/com/ss/ita/rozetka/test/FilterFunctionalityTest.java [hunk 4]
```diff
@@ -138,4 +136,51 @@ public void verifyThatPriceFilterIsCorrect() {
         }
         softAssertion.assertAll();
     }
+
+    @Test
+    @Description("Verify that after selecting and discarding 2 filter options product type page presented correctly")
+    @TmsLink(value = "LVTAQC630-66")
+    public void verifyThatDiscardButtonWorkCorrectly() {
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/test/FilterFunctionalityTest.java:143]: ...Works...

File: src/test/java/com/ss/ita/rozetka/test/FilterFunctionalityTest.java [hunk 5]
```diff
@@ -138,4 +136,51 @@ public void verifyThatPriceFilterIsCorrect() {
...
+
+        assertThat(productTypePage.getProductTitle(1))
+                .as("Title should contains Dell")
+                .contains(company);
+
+        filterSideBar
+                .getFilter(SELLER)
+                .selectOption(seller);
+
+        assertThat(filterSideBar.getFilter(PRODUCER).isOptionSelected(company))
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/test/FilterFunctionalityTest.java:165]: ??

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/ss/ita/rozetka/pageobject/utils/PageUtil.java (hunk 3 above)
Comment (anastasia8755) [reply src/main/java/com/ss/ita/rozetka/pageobject/utils/PageUtil.java:23]: It's PageUtil in this case, we haven't anouther class

File: src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductTypePage.java [hunk 6]
```diff
@@ -132,4 +133,17 @@ public int getProductsQuantity() {
 
         return Integer.parseInt(productsQuantity[1]);
     }
+
+    @Step("ProductTypePage: discard all selected filters")
+    public ProductTypePage discardAllFilters() {
+        $x("//button[@class='catalog-selection__link catalog-selection__link_type_reset']")
+                .shouldBe(Condition.enabled)
+                .click();
+        return new ProductTypePage();
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductTypePage.java:142]: this

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request merged**